Assets/MixedRealityToolkit/Utilities/AssetScriptReferenceRetargeter.cs

```diff
@@ -328,6 +337,11 @@ private static bool IsYamlFile(string filePath)
                             if (!(monoScript is null) && AssetDatabase.TryGetGUIDAndLocalFileIdentifier(monoScript, out string guid, out long fileId))
                             {
                                 Type type = monoScript.GetClass();
+
+                                if (type.Namespace == null || !type.Namespace.Contains("Microsoft.MixedReality.Toolkit"))
+                                {
+                                    Debug.LogErrorFormat("Type {0} is not a member of the Microsoft.MixedReality.Toolkit namespace", type.Name);
+                                }
```

Comment:
We should throw an exception here, as this is a hard stop; we just hit something that is a bug in the MRTK codebase

Reply:
There are currently 9 of these, btw